sgkit/tests/test_preprocessing.py

```diff
@@ -75,31 +87,37 @@ def test_patterson_scaler__raise_on_partial_fit(allele_counts):
 
 
 @pytest.mark.parametrize("dtype", ["i1", "i2", "f2", "f4", "f8"])
-@pytest.mark.parametrize("use_nan", [True, False])
+@pytest.mark.parametrize("use_nan", [False])
```

Comment:
How come we're not testing the nan path here?

Reply:
Ah that's a mistake - I introduced it while debugging. Will revert - thanks for spotting it!